src/features/FreeRooms/Rooms.ts

```diff
@@ -0,0 +1,108 @@
+const ROOMS = [
```

Comment:
Again, scraping this instead of hard-coding values is probably one of the next steps.